src/main/java/perficient/academic/spring5webapp/controllers/CourseController.java

```diff
@@ -0,0 +1,52 @@
+package perficient.academic.spring5webapp.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```

Comment:
We can check the imports here, use * is a bad practice, tomorrow we can check you IDLE configuration X2